docs/admin/serverless/servicemesh/README.md

```diff
@@ -80,7 +80,39 @@ ingress : |- {
 }
 ```
 
-## Deploy InferenceService with Istio sidecar injection
+## Turn on strict TLS on the entire service mesh {#mesh-wide-mtls}
+In the previous section, turning on strict mTLS on a namespace is discussed. Users requiring to lock down all workloads in the service mesh, Istio can be configured with [strict mTLS on the whole mesh](https://istio.io/latest/docs/tasks/security/authentication/mtls-migration/#lock-down-mutual-tls-for-the-entire-mesh).
```

Comment:
```suggestion
In the previous section, turning on strict mTLS on a namespace is discussed. Users requiring to lock down all workloads in the service mesh, Istio can be configured with [strict mTLS on the whole mesh](https://istio.io/latest/docs/tasks/security/authentication/mtls-migration/#lock-down-mutual-tls-for-the-entire-mesh).
```

Users requiring, or users are required?

Reply:
`For users requiring` .... missing `For` :-)

Reply:
...either that, or change like:
Users requiring to lock down all workloads in the service mesh can configure Istio with strict mTLS on the whole mesh.
```

Not sure what's better...